big-practice/src/assets/css/update-customer.css

```diff
@@ -0,0 +1,174 @@
+.modal-update {
```

Comment:
Is this class reused for the `add new` modal?

Reply:
Yes, I will update the class naming suitable to be reused for the add new customer modal